### PR TITLE
My Idea of Automation "Organising Files"

### PR DESCRIPTION
**Pull request by MihirBhave**
This program Organises the files from a specific folder and arranges them in the folders provided by the end user ... Please check it!

Issue: #1

### CODE REVIEW

File: File_Organiser/main.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import os,sys
+import json
+import shutil
+import time
```
Comment (amanBhawsar) [inline File_Organiser/main.py:4]: Please add a readme to this folder for additional libraries which need to be installed like tqdm, colorama etc.
The readme would also be helpful to guide user how to use this code. Like run main.py using python3 main.py.
What are the inputs and those stuff.
It is not clear from **Lets configure ...** then What inputs are to be put i Document and others. The readme should be like one don't need to open your code but is able to run directly following the  steps in readme.

**Review by amanBhawsar — CHANGES_REQUESTED** [review]
@MihirBhave  I checked your code and ran it. It's awesome. Please check the comments and raise PR.
Keep Coding

File: File_Organiser/main.py (hunk 1 above)
Comment (MihirBhave) [reply File_Organiser/main.py:4]: Yes Sir ...  I will do these changes in 2-3 days ..
Thank you for your constructive feedback!

**Comment by MihirBhave** [comment]
Good day @amanBhawsar ... I have made the changes in #5  pull request!

**Comment by amanBhawsar** [comment]
closing it as code is merged from other PR.

**Pull request closed without merging**